crud/common/utils.lua

```diff
@@ -49,6 +50,10 @@ function utils.flatten(object, space_format)
             end
         end
 
+        if bucket_id ~= nil and field_format.name == 'bucket_id' then
```

Comment:
a nit: since we already have `system_fields` conception, maybe it can be

```lua
function utils.flatten(object, space_format, system_values)
    system_values = system_values or {}
    ...
    if system_values.bucket_id ~= nil and field_format.name == 'bucket_id' then ...
    -- if sometimes we have more system fields, it can be done in a cycle
    ...
end
```